frontend/components/Footer/index.js

```diff
@@ -200,36 +202,38 @@ const FooterLinks = ({ links }: Props) => {
     const linkGroups = links.map(linkGroup => {
         const ls = linkGroup.map(l => (
             <li key={l.url}>
-                <a href={l.url}>{l.title}</a>
+                <a className={footerLink} href={l.url}>
+                    {l.title}
+                </a>
             </li>
         ));
 
         return <ul>{ls}</ul>;
     });
 
-    return <div className="footer-links">{linkGroups}</div>;
+    return <div className={footerList}>{linkGroups}</div>;
 };
 
 export default () => (
-    <Footer>
-        <Main className="footer-inner">
+    <footer className={footer}>
+        <Main className={footerInner}>
```

Comment:
There should only be one `<main>` element per page, and it shouldn't be a descendent of a `<footer>`

http://w3c.github.io/html/grouping-content.html#the-main-element